core/src/main/scala/tech/beshu/ror/boot/RorInstance.scala

```diff
@@ -193,12 +204,20 @@ object RorInstance {
     final case class EngineReloadError(underlying: IndexConfigReloadError) extends ScheduledReloadError
   }
 
+  sealed trait TestConfig
```

Comment:
maybe we can move the definition to companion object of RorInstance class?

Reply:
It's already in RorInstance companion object